fecgov/fec-cms
# moved i-graph icon to proper place

## Summary (required)

Moved `fec/static/img/i-graph--primary-contrast.svg` → `fec/fec/static/img/i-graph--primary-contrast.svg`

### Required reviewers

One front end

## How to test

- Check to make sure that the i-graph--primary-contrast.svg was moved to `fec/fec/static/img/i-graph--primary-contrast.svg`.
- Make sure that `i-graph--primary-contrast.svg` is no longer checked into the `/fec/static/img/` directory.
